app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,71 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+/*------------------Signup page css starts-------------------------------*/
+.sigin-container {
+  border: 2px solid #000;
+  margin: 5% auto;
+  padding: 20px;
+  width: 60%;
+  .col-sm-6{
```

Comment:
Thanks for fixing styles and making it a bit consistent, but there is one more minor thing in code that I see not only in styles - empty space between and around `{}`. Somewhere you have it, and somewhere not.